test/integration/dd-agent/serverspec/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require 'serverspec'
+
+# # :backend can be either :exec or :ssh
+# # since we are running local we use :exec
+set :backend, :exec
+
+# require 'serverspec'
+# require 'pathname'
+# require 'net/ssh'
+
+# RSpec.configure do |config|
```

Comment:
Is there any reason to leave this commented out code here?

Reply:
Nope, remains of one of my tries, will clean